distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -116,6 +116,17 @@ Data can be ingested via ${catalog-ui}.
 . Drag and drop file(s) or click to open a navigation window.
 . After selecting the file(s) to be uploaded, select *Start* to begin uploading.
 
+Files are processed individually with a visual cue indicating the completion of each upload.
+If there are any failures, the user will be notified with a message on that specific product.
+More information about the ingest operations can be found in the ingest log.
```

Comment:
More information is available about failed uploads in the ingest log. To say more information about "the ingest operations" sounds like there is another section of the manual that talks more about ingesting in general.